charger/peblar.go

```diff
@@ -84,9 +84,6 @@ func NewPeblar(uri string, id uint8) (api.Charger, error) {
 		return nil, err
 	}
 
-	log := util.NewLogger("peblar")
```

Comment:
Logger still makes sende to see whats going on.

Reply:
> Logger still makes sende to see whats going on.

You mean for future debugging? I can revert it, but currently there is not any logging line in the code so this would be un-used.